core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -195,10 +196,19 @@ private fun dateFormatted(publishDate: Instant): String {
 }
 
 @Composable
-fun NewsResourceDate(
-    publishDate: Instant
+fun NewsResourceDateAndType(
```

Comment:
Since you're using the term "meta data" elsewhere to refer to the information being displayed by this composable, a better name for this might be `NewsResourceMetaData`

Reply:
good catch, updated